ophyd/signal.py

```diff
@@ -158,6 +156,7 @@ def _repr_info(self):
 
     def get(self, **kwargs):
         '''The readback value'''
+        logger.info('%s get() returning ', self, self._readback)
```

Comment:
```suggestion
        logger.info('%s get() returning %s', self, self._readback)
```

Reply:
Fair, but I think we'll just remove this logging call. Too much noise.